File: b/Processor.php
<?php declare(strict_types=1);

class Processor
{
	private string $name;
	private int $price;

	public function __construct(string $name, int $price)
	{
		$this->name = $name;
		$this->price = $price;
	}

	public function setName(string $name): void { $this->name = $name; }
	public function setPrice(int $price): void { $this->price = $price; }

	public function getName(): string { return $this->name; }
	public function getPrice(): int { return $this->price; }

	public function printProcessor()
	{
		echo "Nama Processor : " . $this->name . "<br>";
		echo "Harga Processor : " . $this->price . "<br>";
	}
}

?>

File: b/Processor.java
class Processor
{
	private String name;
	private int price;

	// construct
	public Processor(String name, int price)
	{
		this.name = name;
		this.price = price;
	}

	// defalut construct
	public Processor(){}

	public void setName(String name) { this.name = name; }
	public void setPrice(int price) { this.price = price; }
	
	public String getName() { return this.name; }
	public int getPrice() { return this.price; }

	public void printProcessor()
	{
		System.out.println("Nama Processor : " + this.name);
		System.out.println("Harga Processor : " + this.price);
	}
}